x-pack/legacy/plugins/siem/server/lib/events/elasticsearch_adapter.ts

```diff
@@ -215,7 +214,7 @@ export const formatEventsData = (
 export const formatTimelineData = (
   dataFields: readonly string[],
   ecsFields: readonly string[],
-  hit: EventHit,
+  hit: EventHit | SearchHit,
```

Comment:
Is this being used? In the only usage of `formatTimelineData` I'm seeing `hits` coming back as `EventHit`s. Was there an issue here?